packages/playground-ui/src/components/ui/elements/buttons/button.tsx

```diff
@@ -2,36 +2,35 @@ import { cn } from '@/lib/utils';
 import React from 'react';
 
 export interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
-  as?: React.ElementType;
   className?: string;
-  href?: string;
-  to?: string;
-  prefetch?: boolean | null;
   children: React.ReactNode;
   variant?: 'primary' | 'outline' | 'ghost';
-  target?: string;
 }
 
-export const Button = ({ className, as, variant = 'outline', ...props }: ButtonProps) => {
-  const Component = as || 'button';
+export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
+  ({ className, variant = 'outline', ...props }: ButtonProps, ref) => {
+    return (
```

Comment:
I think we still need the "as" and "href" so that we can have links looking like buttons?